Sources/TuistCore/Models/DestinationPlatform.swift

```diff
@@ -0,0 +1,23 @@
+import Foundation
+import XcodeGraph
+
+/// Represents a platform either for a simulator or a device
+public enum DestinationPlatform: Codable, Equatable {
+    case simulator(Platform)
+    case device(Platform)
+}
+
+extension DestinationPlatform {
```

Comment:
Would it make sense to have this function in the declaration of the `enum` to ease the work for the compiler?